web/src/main/java/org/cbioportal/web/DataAccessTokenController.java

```diff
@@ -88,7 +79,13 @@ private void initializeUsersWhoCannotUseTokenSet() {
                                     @RequestParam(required = false) Boolean allowRevocationOfOtherTokens) throws HttpClientErrorException {
         HttpHeaders responseHeaders = new HttpHeaders();
         responseHeaders.setContentType(MediaType.APPLICATION_JSON);
```

Comment:
I think this happens automatically when your controller is annotated with `@RestController`, so you can delete it from all endpoints.